src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/help/Export-AzureAnalysisServicesInstanceLog.md

```diff
@@ -0,0 +1,72 @@
+---
+external help file: Microsoft.Azure.Commands.AnalysisServices.Dataplane.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Export-AzureAnalysisServicesInstance
+
+## SYNOPSIS
+Exports a log from an instance of Analysis Services server in the currently logged in Environment as specified in Add-AzureAnalysisServicesAccount command
+
+## SYNTAX
+
+```
+Export-AzureAnalysisServicesInstanceLog [-Instance] <String> [-OutputPath] <String> [-WhatIf]
+```
+
+## DESCRIPTION
+The Restart-AzureAnalysisServicesInstance cmdlet restarts an instance of Azure Analysis Services server
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\>Export-AzureAnalysisServicesInstanceLog
+Instance: testserver
+OuptutPath: C:\path\to\log\testserver.log
+```
+
+This command will fetch log from the server 'testserver' in the environment specified in the Add-AzureAnalysisServicesAccount command
```

Comment:
@AOrazaev you may want to update this description of the example to include something about the log being exported to the given path